common.sh

```diff
@@ -32,6 +48,23 @@ get_svn_revision() {
   [ ! -e $TO_DIR ] && svn co -r$SVN_REVISION $SVN_REPO $TO_DIR
 }
 
+build_afl() {
+  $CC $CFLAGS -c -w $AFL_SRC/llvm_mode/afl-llvm-rt.o.c
+  $CXX $CXXFLAGS -std=c++11 -O2 -c $LIBFUZZER_SRC/afl/*.cpp -I$LIBFUZZER_SRC
+  ar r $LIB_FUZZING_ENGINE *.o
+  rm *.o
+
+  BINARY_NAME_EXT="_${FUZZER}"
+}
+
 build_libfuzzer() {
   $LIBFUZZER_SRC/build.sh
+  #mv libFuzzer.a $LIB_FUZZING_ENGINE # more consistent style, breaks backwards compatibility
+  LIB_FUZZING_ENGINE="libFuzzer.a"
```

Comment:
maybe don't do this, instead change all the build.sh files to use $LIB_FUZZING_ENGINE